### ISSUE TITLE
Can't make it work on Linux Mint 18.1

### ISSUE THREAD

**Issue by leninberg** [jepler/cropgui#30]
Hi, I can't use the program. I installed it 

./install.sh -f gtk -p /usr/local/

When it opens, and then go to choose 1 image, it displays this error.

```
Stepconf encountered an error.  The following information may be useful in troubleshooting:

Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/filechooser.py", line 28, in update_preview_cb
    pixbuf = gtk.gdk.pixbuf_new_from_data(i.tostring(),
  File "/usr/lib/python2.7/dist-packages/PIL/Image.py", line 695, in tostring
    "Please call tobytes() instead.")
Exception: tostring() has been removed. Please call tobytes() instead.
```

I press OK (chosing the image to open), I press open, and no image is displayed
Instead, I get this error

```
Stepconf encountered an error.  The following information may be useful in troubleshooting:

Traceback (most recent call last):
  File "/usr/bin/cropgui", line 164, in do_render
    pixbuf = gtk.gdk.pixbuf_new_from_data(rendered.tostring(),
  File "/usr/lib/python2.7/dist-packages/PIL/Image.py", line 695, in tostring
    "Please call tobytes() instead.")
Exception: tostring() has been removed. Please call tobytes() instead.
```

Thanks, I am on Linux Mint 18.1 64 bits

**Comment by jwilk** [comment]
I believe this was fixed in b664d1fefd7140ec4ec37b3b90a9007839029d8a.

**Comment by nicolas-raoul** [comment]
Same error with the version downloaded from https://emergent.unpythonic.net/01248401946

Using the version downloaded from Github solves the problem.
